app/screens/CreateChannelScreen.tsx

```diff
@@ -16,12 +16,13 @@ interface CreateChannelScreenProps
   extends NativeStackScreenProps<AppStackScreenProps<"CreateChannel">> {}
 
 export const CreateChannelScreen: FC<CreateChannelScreenProps> = observer(
-  function CreateChannelScreen() {
+  function CreateChannelScreen({ route }: { route: any }) {
     const pool = useContext(RelayContext) as NostrPool
     const channelManager: ChannelManager = new ChannelManager(pool)
     const formikRef = useRef(null)
 
-    const isPrivate = false
+    // route params
+    const { isPrivate } = route.params
```

Comment:
this breaks other flows.   route may not be present.  

have to be careful with this !

see: 

`
app/screens/ChannelsScreen.tsx:              <Pressable onPress={() => navigation.navigate("CreateChannel")}>
`

please check all deps